### ISSUE TITLE
Could you adda a statistic test to grouping columns

### ISSUE THREAD

**Issue by raffdoc** [haozhu233/ezsummary#10]
Hi, 
This package is what I was searching fro many year, a publication ready table creation using an R code. 
I was wandering if there is a way to add a statistics test to two columns by your grouping variable?
Many thanks beforehand

**Comment by haozhu233** [comment]
Yeah, this feature will definitely be helpful. Right now, the package needs some extra maintenance and  documentation. ;)
